### PR TITLE
[Executorch][Quantization] Backend Config for functional embedding

### PR DESCRIPTION
**Pull request by JacobSzwejbka**
Summary: title

Test Plan: ci

Differential Revision: D42643985

### CODE REVIEW

File: torch/ao/quantization/backend_config/executorch.py [hunk 1]
```diff
@@ -268,6 +268,12 @@ def _get_embedding_op_configs() -> List[BackendPatternConfig]:
                 .set_root_module(embedding_op)
                 .set_reference_quantized_module(ref_embedding_op)
                 ._set_input_output_observed(False))  # This is temporary, and will be removed soon
+        #config for functional embedding
+        embedding_op_configs.append(
+            BackendPatternConfig(torch.nn.functional.embedding)
+                .set_observation_type(ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT)  # noqa: E131
+                .set_dtype_configs(dtype_configs)
```
Comment (jerryzh168) [inline torch/ao/quantization/backend_config/executorch.py:275]: is this correct?

Comment (JacobSzwejbka) [reply torch/ao/quantization/backend_config/executorch.py:275]: I think so?

**Review by jerryzh168 — APPROVED** [review]

**Pull request closed without merging**